src/components/datagrid/data_grid_body.tsx

```diff
@@ -163,30 +158,6 @@ export const EuiDataGridBody: FunctionComponent<
     return rowMap;
   }, [sorting, inMemory, inMemoryValues, schema, schemaDetectors]);
 
-  const setCellFocus = useCallback(
```

Comment:
This was meant to manage the distinction between `focusedCell`'s value at the top *data_grid.tsx* (focusedCell is the column,row tuple where the values match the visible rows, but *data_grid_row.tsx* treated `focusedCell`'s row on the row index instead. Row & Cell have now been provided the visible row index in addition to their logical row index, making this code block & its mapping obsolete.